vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -2024,3 +1982,12 @@ func TestStandardExecutorTransformSubnetTx(t *testing.T) {
 		})
 	}
 }
+
+func defaultTestConfig(latestForkTime time.Time) *config.Config {
```

Comment:
i think this function would be more readable if we used the fork constants: defaultTestConfig(durango, time.Time)

Reply:
Done, at the cost of adding the `//nolint: unparam` clause since we always pass `durango` here